composer.json

```diff
@@ -5,7 +5,8 @@
     "license": "proprietary",
     "require": {
         "php": ">=7.4",
-        "spryker-sdk/spryk": "^0.3.1",
+        "spryker-sdk/spryk": "^0.4.0",
+        "spryker-sdk/spryk-src": "dev-master",
```

Comment:
U dont need this one imo

Reply:
Spryk-gui uses some classes from spryk-src
for example https://github.com/spryker-sdk/spryk-gui/blob/master/src/SprykerSdk/Zed/SprykGui/SprykGuiDependencyProvider.php#L13

Reply:
It shouldnt, as that one is becoming the phar file of spryker-sdk/spryk
cc @stereomon 

also, dev master would never be allowed here anyways.
So we need a different way of using those.